12_listcomprehension_lab/01.py

```diff
@@ -1,8 +1,13 @@
-"""
-Use range() and list comprehension to get
-the list of all lowercase english letters
-Hint: look for chr() and ord()
-"""
-
-
-
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 2
+
+"""
+
+word = [chr(i) for i in range(97,123)]

```

Comment:
Do you really remember that chr(97) == 'a'? Do you expect everyone reading your code to remember? Better to use ord('a') and ord('z') as the limits: `range(ord('a'), ord('z')+1)`